suites/quincy/rbd/tier-2_rbd_trash_purge.yaml

```diff
@@ -0,0 +1,97 @@
+# Tier2: Extended RBD acceptance testing
+#
+# This test suite runs addition test scripts to evaluate the existing functionality of
+# Ceph RBD component.
```

Comment:
Can you be more specific on the tests being done in this test suite?

Reply:
Done.